app/helpers/pipeline_outputs_helper.rb

```diff
@@ -190,4 +190,6 @@ def status_display_helper(states_by_output_hash, results_finalized_var)
       "HOST FILTERING"
     end
   end
+
+  module_function :get_s3_file
```

Comment:
is this for the purpose of avoiding the `include PipelineOutputsHelper`?

Reply:
It's so I can stub the method in the test — otherwise, it says that `PipelineOutputsHelper` doesn't have the method

Reply:
There are some alternatives: 
1. Make the method static, if safe
2. Mock the S3 service: 
	``` 
		s3 = Aws::S3::Client.new(stub_responses: true)
      	s3.stub_responses(:get_object,
	```
3. Mock any instance with `allow_any_instance_of(PipelineOutputsHelper).to receive(: get_s3_file).and_return(step_status_data.to_json)`

Since this looks like an integration test... I would go with option 2.

Reply:
I went with the second option (should be reflected in the code now)